tests/unit/test_operator.py

```diff
@@ -451,12 +461,41 @@ def test_send_manifests(self, create_manifests: MagicMock, harness: Harness):
     )
     @patch("charm.MlflowCharm._get_object_storage_data")
     @patch("charm.MlflowCharm._get_relational_db_data")
-    def test_on_event(
+    def test_on_event_wainting_for_exporter(
+        self,
+        get_relational_db_data: MagicMock,
+        get_object_storage_data: MagicMock,
+        harness: Harness,
+    ):
+        get_object_storage_data.return_value = OBJECT_STORAGE_DATA
+        get_relational_db_data.return_value = RELATIONAL_DB_DATA
```

Comment:
do not remember, can return value be set directly in the patch wrapper ?

Reply:
You right. I refactored. Thanks :)